app/views/authors/toc.html.haml

```diff
@@ -60,7 +60,13 @@
                   != "&nbsp;&nbsp;&nbsp;"
                   %span.static-btn
                     %b= link_to t(:edit_metadata), authors_edit_path(id: @author.id)
-        
+
+                  != '&nbsp;&nbsp;&nbsp;'
+                  %span.static-btn
+                    %b= link_to 'Refresh uncollected works',
```

Comment:
Temporary link to call service to refresh uncollected works collection. I did not bothered about I18n, as it is a temporary solution.